src/testcafe.js

```diff
@@ -47,6 +52,17 @@ export default class TestCafe {
         });
     }
 
+    _setupSourceMapsSupport () {
+        sourceMapSupport.install({
+            hookRequire:              true,
+            handleUncaughtExceptions: false,
+            environment:              'node'
+        });
+    }
+
+    _registerErrorHandlers () {
```

Comment:
Don't need an addition function. Just call `registerErrorHandlers()` in constructor.